tests/unit/kernel/PeriodicKernel_unittest.cc

```diff
@@ -0,0 +1,137 @@
+#include <shogun/kernel/PeriodicKernel.h>
+#include <shogun/lib/SGMatrix.h>
+#include <shogun/features/DenseFeatures.h>
+#include <gtest/gtest.h>
+
+using namespace shogun;
+
+TEST(PeriodicKernelTest,test_kernel_matrix)
+{
+	// Construct matrix with features
+	SGMatrix<float64_t> matrix(2,3);
+    for (int32_t i=0; i<6; i++) {
+        matrix.matrix[i]=i;
+    }
+
+    // Load them into DenseFeatures
+    CDenseFeatures<float64_t>* features= new CDenseFeatures<float64_t>();
+    features->set_feature_matrix(matrix);
+
+    // Construct kernel and compute kernel matrix
+	CPeriodicKernel* kernel = new CPeriodicKernel(features, features, 1.0, 5.0);
+	SGMatrix<float64_t> computed_kernel_matrix = kernel->get_kernel_matrix();
+
+	// Define expected kernel matrix
+	SGMatrix<float64_t> expected_kernel_matrix(3,3);
+	expected_kernel_matrix(0,0) = 1.0;
+	expected_kernel_matrix(1,0) = 0.14718930341788436;
+	expected_kernel_matrix(2,0) = 0.724874286159619241;
+	expected_kernel_matrix(0,1) = 0.14718930341788436;
+	expected_kernel_matrix(1,1) = 1.0;
+	expected_kernel_matrix(2,1) = 0.14718930341788436;
+	expected_kernel_matrix(0,2) = 0.724874286159619241;
+	expected_kernel_matrix(1,2) = 0.14718930341788436;
+	expected_kernel_matrix(2,2) = 1.0;
+
+	EXPECT_EQ(true, computed_kernel_matrix.equals(expected_kernel_matrix)) 
+		<< "Computed kernel matrix did not match expected matrix.";
+
+	SG_SDEBUG("EXPECTED:\n");
+	SGMatrix<float64_t>::display_matrix(expected_kernel_matrix);
```

Comment:
Could you remove those prints? they pollute the std output when running things on the buildbt